### DISCUSSION TITLE
Update axios to `1.6.0` as its vulnerable currently

### DISCUSSION THREAD

**Discussion by TomPeas** [graphql-hive/console#3229] [Feature request]
The version of axios used by `@graphql-hive/client` is vulnerable: https://security.snyk.io/vuln/SNYK-JS-AXIOS-6032459

**Comment by dotansimha** [comment]
Thanks @TomPeas , I'm locking this one in favor of https://github.com/kamilkisiela/graphql-hive/issues/3232
